tasks/allocs/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "oxc_allocs"
```

Comment:
`allocs` is too vague, maybe `oxc_track_memory_allocations` and `track_memory_allocations` for the directory name.

You can add a just command to shorten typing: `just allocs`.

Reply:
Fair. I added the `just allocs` alias, but renamed the task to `oxc_track_memory_allocations`.